### DISCUSSION TITLE
cargo install doesn't install latest version

### DISCUSSION THREAD

**Discussion by bobblkabb** [j0ru/kickoff#155] [Q&A] [answered]
Hi,

I wonder why cargo install does install 0.72 instead of 0.74?

**Comment by j0ru** [comment, marked as answer]
Hi,
I had forked a library that caused about half of the compile time, and set the dependency to my repo. Unfortunately, I had to learn that crates.io doesn't accept packages that have dependencies outside of crates.io.
Because of that, there were no crates releases since then, but I had to reverse that either way, since it seems to have caused trouble with the AUR. Just hadn't realized that this would also solve the crates problem, the newest version should be up now.

For now I either have to find a way to make the dependency obsolete, or wait for the upstream project to accept my PR, but the project has been dormant for the last 5 years :/
https://github.com/divvun/xkb-parser/pull/2
